WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1217,6 +1217,11 @@ ComposeViewHtmlEditor       = padding:@ComposeViewOutsidePadding@; background-co
 ComposeViewPlainTextEditor  = @FontSize-big@ @FontFamily-fixed@ @FullWidth@ @AppBorder@ overflow:auto; padding:8px;
 ComposeViewHeaderEmptyRight = padding-right: 18px;
 
+AttachBriefcaseItem         = border-style:solid; border-width:0 0 1px 0; border-color:transparent;
+AttachBriefcaseItemSelected = border-bottom-color:@PanelBorderColor@;
+AttachBriefcaseItemChild    = padding-left:@MailViewInnerSpacing@;
+AttachBriefcaseItemFirstChild = padding-left:@SkinWrapperPadding@;
```

Comment:
Same suggestions as above. Again feel free to keep as it is, if you feels that okay.

Thanks